releasenotes/notes/lo-freqs-circuit-c2ec161873eb4a68.yaml

```diff
@@ -0,0 +1,29 @@
+---
+features:
+  - |
+    The ability to pass qubit and measurement LO frequencies has now been added to circuit jobs. You
```

Comment:
```suggestion
    The ability to pass default qubit and measurement LO frequencies has now been added to circuit jobs. You
```

Reply:
The only thing about the word "default" is that these will be the ones used in the job for circuits. In the pulse case you can override them on a per-schedule basis.

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6167/commits/705ca3c97ae84c6f950f430f1eea576d4eb93e1c.